Create ui-c8y-1023-58-3-correct-agents-resolving-in-none-mono-repo-build-11431-graft-release.md (#4410)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-58-3-correct-agents-resolving-in-none-mono-repo-build-11431-graft-release.md

---
date: ""

> title: Correct AI agents resolving

Reviewer: ```suggestion
title: AI agents are now properly resolved
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66253
version: 1023.58.3
---

> If the AI Agent Manager was configured and an provider API key was set, the agents still were not resolved correctly, which could cause runtime issues when using the AI Code assistant. This has been fixed so that agents are now properly resolved.

Reviewer: ```suggestion
Previously, when the AI Agent Manager was configured and a provider API key was set, the agents were still not resolved correctly, which could cause runtime issues when using the AI Code assistant. This has been fixed so that agents are now properly resolved.
```
